exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/columnreaders/AsyncPageReader.java

```diff
@@ -409,9 +414,10 @@ public Void call() throws IOException {
 
       DrillBuf pageData = null;
       timer.reset();
-      try {
+      parqProcessing: try {
```

Comment:
Good point. I didn't realize that the `finally` block is empty. If there was something to be done, a labelled break would allow the finally to be executed.